### ISSUE TITLE
stylelintrc rules not applied

### ISSUE THREAD

**Issue by ovitrif** [dmnsgn/sublime-stylefmt#14]
It doesn't read the rules inside stylelintrc.

The extension for VS Code does.

**Comment by dmnsgn** [comment]
Is this still the case with the latest version?

**Comment by igregson** [comment]
From my testing I can also confirm that this bug persists with the latest version.

**Comment by benfrain** [comment]
Running v5 of this plugin and I'm not seeing it pick up the .stylelintrc files either (running Stylelint 7.5.0)

**Comment by dmnsgn** [comment]
@benfrain Is there any log in the sublime console when you `Run Stylefmt`?

**Comment by benfrain** [comment]
No, plugin itself loads without issue too (no errors in console and does the default formatting of putting space before `@media` rules and the like).

**Comment by benfrain** [comment]
Here's my .stylelintrc https://gist.github.com/benfrain/aea885ad0dfcbe21207734c1deea0707

**Comment by dmnsgn** [comment]
I have tried with your `.stylelintrc` file and changed `"indentation": 16,` and I could see the 16 spaces applied.

ST version or folder structure is the only thing I can think of to be a possible reason.

**Comment by benfrain** [comment]
Thanks. Turns out it was a combination of other factors. I needed to get the latest Stylelint and update my .stylelintrc accordingly. One additional issue was that Stylefmt adds its own standard config on top of the rules in the .stylelintrc (which causes a lot of confusion).
As far as I can tell there is an options flag `--onlyStylelint` [ref: https://github.com/morishitter/stylefmt/issues/86] - can this be passed via this plugin?
